### DISCUSSION TITLE
Support string value in prometheus reporting

### DISCUSSION THREAD

**Discussion by gibix** [savonet/liquidsoap#2253] [User Support]
**Is your feature request related to a problem? Please describe.**
I would like to report which track is playing of a playlist with some metadata attached. But at now the prometheus apis support only float values. 

**Describe the solution you'd like**
The summary should support string type.

**Additional context**
```
track_metric = prometheus.summary(labels=["source"], help="witch track is played in source", "track_metric")

def log_playlist(m) =
		ignore(track_metric(label_values=[m{'source'], m['filename']))
end

my_playlist.on_metadata(log_playlist)
```

**Comment by toots** [comment]
I believe that prometheus does not support string values. You can, however, emulate that with labels. See for instance: https://stackoverflow.com/questions/65850083/prometheus-java-client-export-string-based-metrics/65881775#65881775
